src/Stratis.Bitcoin.Features.Api/dashboard_static_files/index.html

```diff
@@ -0,0 +1,42 @@
+﻿<!DOCTYPE HTML>
+<head>
+    <title>Stratis Dashboard</title>
+    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.8.0/jquery.min.js"></script>
```

Comment:
Simple example of how html/javascript static files can render a dashboard.

Reply:
How come jquery 1.8.0 is being used and not the latest? There are continuously found various exploits in older versions of jquery that are fixed, and 1.8.0 includes the 11974 issue reported here, which has a lot of XSS issues: https://bugs.jquery.com/ticket/11974

Reply:
@StratisDeveloper yes please upgrade and it should be as simple as updating the nuget